server/src-lib/Hasura/Tracing.hs

```diff
@@ -77,9 +80,9 @@ instance HasReporter m => HasReporter (ExceptT e m) where
 -- the active span within that trace, and the span's parent,
 -- unless the current span is the root.
 data TraceContext = TraceContext
-  { tcCurrentTrace  :: Word64
-  , tcCurrentSpan   :: Word64
-  , tcCurrentParent :: Maybe Word64
+  { tcCurrentTrace   :: Word64
```

Comment:
Our [style guide](https://github.com/hasura/graphql-engine/blob/master/server/STYLE.md#data-types-1) recommends using strict fields for ADTs. We should add strictness annotations to these.

Reply:
Fixed it, Thanks :)